app/views/devise/shared/_errors_massages.html.slim

```diff
@@ -0,0 +1,11 @@
+.alert.alert-danger.alert-dismissible.mb-2[role="alert"]
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
+    span[aria-hidden="true"]
+      | ×
+  h5.alert-heading.mb-2
+    | Warning!
+  p
+    = error_sentence
+  - error_messages.each do |msg|
```

Comment:
and actually you can use code like
= f.error_messages

USE IT to output errors.